extensions/scheduler/runtime/src/main/java/io/quarkus/scheduler/Scheduled.java

```diff
@@ -39,6 +39,16 @@
 @Repeatable(Schedules.class)
 public @interface Scheduled {
 
+    /**
+     * Optionally defines a unique identifier for this job.
+     * <p>
+     * If the value is not given, Quarkus will generate a unique id.
+     * <p>
+     * 
+     * @return the unique identity of the schedule
+     */
+    String identity() default "";
```

Comment:
We should also use `Scheduled.identity()` in the `SimpleScheduler`.

Reply:
Yes, I forgot about that one. Should be good now.